src/Areas/AppConfig/Options/KeyValue/KeyValueShowOptions.cs

```diff
@@ -1,6 +1,10 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-namespace AzureMcp.Options.AppConfig.KeyValue;
+
```

Comment:
Duplicate header here?  Are you running the eng/scripts/AddCopyrightHeaders.ps1?